e2e-tests/tests/demand-backup/14-assert.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+timeout: 30
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: 06-read-data-gcp-0
+data:
```

Comment:
why do we use this one?

Reply:
Because I want to check that data is actually deleted from all nodes and since we used the same approach for checking that data exists I used the same here. Alternative would be to not use the assert file, but just make a query and check in the same script where we delete data, but somehow I felt that it is not in the spirit of this framework.